### PR TITLE
update fields and graphs for GPS Rescue 4.4

### PR DESCRIPTION
**Pull request by ctzsnooze**
The recent fields update doesn't work because flightlog_fielddefs.js was not updated, so `GPS_RESCUE_HEADING` and `GPS_RESCUE_HEADING` logs were not displaying correct debug or field names.

Also the `GHST_MSP` and `RX_STATE_TIME` debug fields were not considered.

Works properly now with `GPS_RESCUE_HEADING`, `GPS_RESCUE_TRACKING` and `GPS_RESCUE_HEADING` debugs.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by toddferrante** [comment]
Windows: Be sure to uninstall previous version before installing this one.  On my first try I installed the new version over the old one.  This causes the software to hang and not start up.  Needed to kill it in the task manager.

**Comment by haslinghuis** [comment]
@toddferrante having rebased https://github.com/betaflight/blackbox-log-viewer/pull/591 please test

**Pull request merged**
